src/__test__/utils/postErrorToSlack.test.js

```diff
@@ -0,0 +1,83 @@
+import postErrorToSlack from 'utils/slack/postErrorToSlack';
+import fetchMock, { enableFetchMocks } from 'jest-fetch-mock';
+
+enableFetchMocks();
+
+const mockError = { stack: 'Mock error stack' };
+const mockInfo = { componentStack: 'Mock component stack' };
+const mockReduxDump = {
+  cellInfo: {
+    focus: {
+      store: 'cellSets',
+      key: 'louvain',
+    },
+    groupedTrack: 'louvain',
+    selectedTracks: [
+      'louvain',
+    ],
+  },
+  cellMeta: {
+    mitochondrialContent: {
+      loading: true,
+      error: false,
+      data: [],
+    },
+    doubletScores: {
+      loading: true,
+      error: false,
+      data: [],
+    },
+  },
+};
+
+jest.useFakeTimers('modern').setSystemTime(new Date('2020-01-01').getTime());
+
+delete window.location;
+window.location = {
+  href: 'http://localhost:3000/experiments/testae48e318dab9a1bd0bexperiment/data-exploration',
+};
+
+jest.mock('@aws-amplify/auth', () => ({
+  currentAuthenticatedUser: () => Promise.resolve({
+    attributes: {
+      name: 'John Doe',
+      email: 'fake@email.com',
+    },
+    username: '5152fake-eb52-474c-user-mocke8c8user',
+  }),
+}));
+
+describe('PostErrorToSlack', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    fetchMock.resetMocks();
+    fetchMock.doMock();
```

Comment:
what does `doMock`?

Reply:
`doMock()` enables `fetchMock`. Actually `resetMock()` only clears out the mock data and does not disable/unmock `fetchMock`, so `doMock` is not needed here. I'll remove this.